src/app/modules/backbone/components/nav-item/nav-item.component.html

```diff
@@ -1,5 +1,5 @@
-<li class="nav-item-container" routerLinkActive="nav-item-active" [ngClass]="{'nav-item-force-open': label == openLabel}">
-    <div class="nav-item" [routerLink]="[link]">
+<li class="nav-item-container" [ngClass]="{'nav-item-force-open': label == openLabel}">
+    <div class="nav-item" [routerLink]="[link]" routerLinkActive="nav-item-active">
         <span>{{ label }}</span>
         <span *ngIf="children.length > 0">
             <svg class="nav-item-arrow" fill="currentColor" viewBox="0 0 20 20">
```

Comment:
We have lost the nesting of the menu items? did it happen in another merge? or is it a result of the change here?

Reply:
This was deliberate. When we were applying the active class to the parent list-item, it highlighted the whole container which extended behind the children and caused an ugly appearance when the children were selected (highlighted in red to make it easier to see):
![image](https://user-images.githubusercontent.com/43872824/124462477-ff690080-dd91-11eb-8346-9c2d830f4c78.png)

By highlighting the inner div, which doesn't include the children or the container which extends behind them, I think the result looks nicer (or, at least, less like a mistake. You may want to change it further, e.g. margins):
![image](https://user-images.githubusercontent.com/43872824/124462600-2aebeb00-dd92-11eb-80b5-2aad4b097ab6.png)

Reply:
I just realised you might be seeing the effect of the parent menu being forced open on activation. Without this, we'd be highlighting a menu item (e.g. products/coffees) that wasn't visible to the user unless they moused over the menu (potentially leading them to think they were looking at all products instead of just coffees, for example).

The little menu arrow also gets hidden when its respective menu gets forced open since I thought it was a little confusing to have an arrow indicating the presence of a mouse-over menu, but the menu was already forced open (meaning the arrow wasn't adding anything, just a redundant—potentially even confusing—detail).

Reply:
Yes, I see what you mean. The dropdown works correctly if I navigate to general-information. So all good